doc/classes/RayCast2D.xml

```diff
@@ -69,13 +69,14 @@
 			<return type="Vector2" />
 			<description>
 				Returns the normal of the intersecting object's shape at the collision point, or [code]Vector2(0, 0)[/code] if the ray starts inside the shape and [member hit_from_inside] is [code]true[/code].
+				[b]Warning:[/b] Check that [method is_colliding] returns [code]true[/code] before calling this method to ensure the returned normal is meaningful and up-to-date.
 			</description>
 		</method>
 		<method name="get_collision_point" qualifiers="const">
 			<return type="Vector2" />
 			<description>
-				Returns the collision point at which the ray intersects the closest object.
-				[b]Note:[/b] This point is in the [b]global[/b] coordinate system.
+				Returns the collision point at which the ray intersects the closest object, in the global coordinate system.
```

Comment:
```suggestion
				Returns the collision point at which the ray intersects the closest object, in the canvas layer coordinate system.
```
As discussed in https://github.com/godotengine/godot-proposals/issues/3866, "global" can be confusing in 2D. Since this PR changes this line, I believe it best to make this more precise.

Reply:
I agree that would be more precise, but it is more confusing if the `RayCast2D` is not under a `CanvasLayer` node, so I think the explanation should cover that (very common) case as well. I know it's layer 0 internally or something like that, but I'm not convinced the user (whose `RayCast2D` is probably not under a `CanvasLayer` node) will know. I didn't plan to go into this here, but if you want you can make a suggestion addressing this.

Reply:
Unfortunately I don't have a better idea - it's a complex topic.
If this isn't feasible for now, it can easily be deferred to a later point in time. I would have considered this just nice to have in the context of this PR.

Reply:
So I agree it could be improved, but I would prefer to leave it for another PR.